docs/gathered-data.md

```diff
@@ -900,18 +989,42 @@ gathers the alerts that have been silenced.
 * See: docs/insights-archive-sample/config/silenced_alerts
 * Id in config: clusterconfig/silenced_alerts
 * Since version:
-  * 4.10+
+  - 4.10+
+
+
+## StorageCluster
+
+Collects `storageclusters.ocs.openshift.io` resources
+from Openshift Data Foundation Stack.
+
+### API Reference:
+https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/storagecluster_types.go
+
+### Sample data
+docs/insights-archive-sample/config/storage/openshift-storage/storageclusters/ocs-storagecluster.json
+
+### Location in archive:
+| Version | Path														|
+| ------- | --------------------------------------------------------	|
+| <= 4.12 | config/storage/{namespace}/{name}.json 					|
+| 4.13+   | config/storage/{namespace}/storageclusters/{name}.json 	|
```

Comment:
This is similar problem as in https://github.com/openshift/insights-operator/pull/704 (it's confusing at least for me) and the `4.13+` has not been released yet at the moment so it can be theoretically dangerous to do such predictions right now.